.github/workflows/run_tests.yml

```diff
@@ -51,11 +51,14 @@ jobs:
       - name: Install package
         run: pip install .[test]
 
+      - name: Install requirements
```

Comment:
I think it would be better to stick to the same way of installing dependencies as for running the tests itself (see lines above, i.e. use `run: pip install .[tools,test]`). In fact we could the just then actually remove the requirements.txt as it is not used in the setup.py

Reply:
So the line you mention just install all package that are require ?

Reply:
Yes, this way of installing takes care of the extras that are required for the `tools` and `test` that are included, see also in the setup.py